en/docs/getting-started/basic-architecture.md

```diff
@@ -0,0 +1,80 @@
+
+#### API Developer Portal
+
+![Developer Portal]({{base_path}}/assets/img/get_started/developer-portal-overview.png)
+
+The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to self-register, discover, evaluate, subscribe to and consume APIs securely and easily.
+
+
+#### API Analytics
+
+![API Analytics]({{base_path}}/assets/img/get_started/analytics.png)
+
+Monitoring and analytics of APIs deployed on WSO2 API Manager are provided by the WSO2 API Manager Analytics component. This component includes different types of dashboards to provide more detailed different statistical views of your APIs. Furthermore, you can secure your system by configuring alerts and notifications on pre-determined events to identify any unusual behaviors in near real-time.  For more information, see Working with Analytics.
+
+
+## Data Plane
+
+Data plane is where the created API is exposed to the public consumers and acts as the proxy for API calls. This also provides additional capabilities such as enforcing security, rate limiting etc. (policy enforcement point).
+
+### Components
+
+#### API Gateway
+
+![API Gateway]({{base_path}}/assets/img/learn/gateway-overview.png)
+
+WSO2 API Gateway acts as the entry point for an API request made to an API managed by WSO2 API Manager.
+
+JWT token validation is done by the API Gateway. The API Gateway validates it's signature and subscription using the information it has in it's an in-memory map. This in-memory map includes API-related information, application-related information, subscription-related information, etc. which is updated each time an artifact (API/application) is updated.
+  
+ Once the token is validated, the API Gateway acts upon the API request before sending it to the backend. It first processes the message to a pre-configured format (e.g. JSON/XML/CSV/etc.).  It then applies security policies, rate-limiting policies,  collects statistics, etc., via its handlers.  The mediators then act upon the API payload based on the mediation logic developed. The message is then formatted to a pre-configured format (e.g.JSON/XML/CSV/etc.) and sent to the backend. The WSO2 API Gateway supports transports such as HTTP, HTTPS, etc. It is also able to scale on-demand in cloud environments and is easily pluggable in non-cloud environments. 
+ 
+## Control Plane
+ 
+ Control plane is where the API security validation, API key generation and rate limiting decisions are taken.
+ 
+### Components
+ 
+#### Key Manager
+ 
+ ![Key Manager]({{base_path}}/assets/img/get_started/key-manager-overview.png)
```

Comment:
The main use-case of the Key Manager is for clients to login and get tokens. These tokens are then sent to the gateway when accessing APIs. This diagram of the Key Manager illustrates an internal architecture of the product where the KM is used by the dev portal and publisher but misses to illustrates the KM's primary use case. I suggest to redraw this image to illustrate the primary use case of the KM and to ignore the product internals to keep it simple.

Reply:
noted. Will do.